app/firebase-emulator/start-emulator.js

```
@@ -0,0 +1,60 @@
+/**
+ * Start Firebase emulator for tests
+ * This script is used by the test framework to start the Firebase emulator
+ */
+
+const { spawn } = require('child_process');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)